entrypoint.sh

```diff
@@ -115,7 +115,7 @@ else
 		svn update --set-depth infinity "tags/$STABLE_TAG"
 
 		# Not doing the copying in SVN for the sake of easy history
-		rsync -c "$TMP_DIR/readme.txt" "tags/$STABLE_TAG/"
+		rsync -c "$GITHUB_WORKSPACE/readme.txt" "tags/$STABLE_TAG/"
```

Comment:
Even though this will likely work for both the `.distignore` and `.gitattributes` methods, I would prefer to keep this precise and use `$GITHUB_WORKSPACE` for when a `.distignore` is present as is done further up in the file, falling back to `$TMP_DIR`. I actually wonder if maybe it's best to set `TMP_DIR=$GITHUB_WORKSPACE` in that if block, a little bit of a trick but that way we don't worry about another conditional here.

Reply:
I have set the variable `TMP_DIR=$GITHUB_WORKSPACE` in the `.distignore` block as that basically reduces conditional usage of source directories below and a single variable can be referred to as a source of truth throughout the conditions.